seventh_ex.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+
+def swap_first_and_last_rows(randomized_array) -> None:
+    """
+    This function gets an array and swaps it's first rows with it's last rows.
+    :param randomized_array: the array that we will swap it's first and last rows.
+    """
+    swapped_array = randomized_array[::-1]
```

Comment:
I don't understand why you aren;t returning anything, what is the purpose of the function?